torchgeo/datamodules/l8biome.py

```diff
@@ -0,0 +1,62 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+"""L8 Biome datamodule."""
+
+from typing import Any
+
+import torch
+
+from ..datasets import L8Biome, random_bbox_assignment
+from ..samplers import GridGeoSampler, RandomBatchGeoSampler
+from .geo import GeoDataModule
+
+
+class L8BiomeDataModule(GeoDataModule):
+    """LightningDataModule implementation for the L8 Biome dataset.
+
+    .. versionadded:: 0.5
+    """
+
+    mean = torch.tensor(0)
+    std = torch.tensor(10000)
+
+    def __init__(
+        self, batch_size: int = 64, num_workers: int = 0, **kwargs: Any
```

Comment:
Need to add batch_size, patch_size, and length parameters, same as the base class. You'll also want to set these in the yaml file to make them smaller for testing purposes